src/test/java/games/strategy/util/UtilTest.java

```diff
@@ -0,0 +1,28 @@
+package games.strategy.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class UtilTest {
+
```

Comment:
"Empty string is a valid argument" While valid, is it ever the case a client would want to, in a valid context, hash an empty string? It almost certainly would be an error on part of the client no?

> will probably 

Hate to say, that level of certainty does not really cut it. Part of the point of having an empty string and null as inputs is to document as a test case what the behavior should be.